PyRDF/CallableGenerator.py

```diff
@@ -62,22 +20,39 @@ def get_callable(self):
         self.root_node.graph_prune()
 
         def mapper(node_cpp, node_py=None):
+            """
+            The mapper callable that recurses
```

Comment:
I would not call it mapper in the documentation, actually at some point we should just name this function `callable`, since there will be no concept of mapper and reducer here, and the `get_reducer_callable` will be completely removed.

Reply:
Same comment applies for the docstring of `get_callable` and the one of `CallableGenerator`, please modify those as well.